src/mm-file-upload/mm-file-upload.scss

```diff
@@ -0,0 +1,18 @@
+/**
+ * @license
+ * Copyright (c) 2015 MediaMath Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at http://mediamath.github.io/strand/LICENSE.txt
+
+*/
+/* mm-file-upload sass */
+@import "_bourbon";
+@import "_color";
+@import "_button";
+@import "_mixins";
+
+:host {
```

Comment:
To get proper metrics for the host, we'd need to add: 
```
position: relative;
display: inline-block;
vertical-align: middle;
```
...right now it's measured as 18px height, even though the fields are 29px.